DataSetExplorerTests/Unit/ImporterTests.cs

```diff
@@ -13,30 +13,37 @@ public class ImporterTests
         [Fact]
         public void Imports_data_set_instances_and_annotations()
         {
-            var dataSet = new DataSet("Test");
+            var codeSmells = new List<CodeSmell>
```

Comment:
How good is our test coverage for the dataset explorer functionality? Something to keep in mind.